tests/api-mongo.test.js

```diff
@@ -388,5 +399,30 @@ describe('MongoDB Words', function () {
           done();
         });
     });
+<<<<<<< HEAD
+});
+
+describe('API Requests For Home Directory "/"', () => {
+    describe('/undefinedRoute', () => {
+      it('should return response status of 404', (done) => {
+        chai.request('http://igboapi.com').get('/undefined').end((_, res) => {
```

Comment:
`http://igboapi.com` should be a constant, you can create the new constant inside the `tests/shared/constants.js` file.